### PR TITLE
P4INFRA-63: Create getJobApplication function

### PR DESCRIPTION
**Pull request by mpa-mxiang**
For this PR, I:
- Created a function to view the job application table using userId to get a list of jobs the user has already applied

### CODE REVIEW

**Review by Mich-Elle-Lo — APPROVED** [review]
Great!

**Comment by mpa-mxiang** [comment]
Discussed @sgaggen and he agreed that this PR is good to merge now, will see if further change need to do for this later after merge.

**Pull request merged**
